ArcGISRuntimeSDKQt_QMLSamples/Layers/ExportVectorTiles/ExportVectorTiles.qml

```diff
@@ -0,0 +1,275 @@
+
+        onCreateDefaultExportVectorTilesParametersStatusChanged: {
+            if (createDefaultExportVectorTilesParametersStatus !== Enums.TaskStatusCompleted)
+                return;
+
+            // Using the reduced fonts service will reduce the download size of a vtpk by around 80 Mb
+            // It is useful for taking the basemap offline but not recommended if you plan to later upload the vtpk
+            defaultExportVectorTilesParameters.esriVectorTilesDownloadOption = Enums.EsriVectorTilesDownloadOptionUseReducedFontsService;
+
+            // Create a path to store the vector tile package
+            const vectorTileCachePath = System.temporaryFolder.url + "/vectorTiles_%1.vtpk".arg(new Date().getTime().toString());
+            // Create a path to store the styling resources (in this case, the night mode version of the layer)
+            const itemResourcePath = System.temporaryFolder.url + "/itemResources_%1".arg(new Date().getTime().toString());
+
+            exportVectorTilesJob = exportVectorTilesWithStyleResources(defaultExportVectorTilesParameters, vectorTileCachePath, itemResourcePath);
+
+            exportVectorTilesJob.resultChanged.connect(() => {
+                if (exportVectorTilesJob.result) {
+                    // Create a vector tiled layer when the download is completed
+                    const exportedVectorTiledLayer = ArcGISRuntimeEnvironment.createObject("ArcGISVectorTiledLayer", {
+                        vectorTileCache: exportVectorTilesJob.result.vectorTileCache,
+                        itemResourceCache: exportVectorTilesJob.result.itemResourceCache
+                    });
+                    const basemap = ArcGISRuntimeEnvironment.createObject("Basemap");
+                    basemap.baseLayers.append(exportedVectorTiledLayer);
+                    map.basemap = basemap;
+                    exportProgressWindow.visible = false;
+                }
+            });
+
+            // Display the download progress to the user
+            exportVectorTilesJob.progressChanged.connect(() => {
+               exportProgress = exportVectorTilesJob.progress;
+            });
+
+            exportVectorTilesJob.statusChanged.connect((status) => {
+                exportJobStatus = status;
+                switch(exportJobStatus) {
+                case 0: // Not started
+                    button.text = "Export area"
+                    break;
+                case 1: // Started
+                    button.text = "Cancel export"
+                    break;
+                case 2: // Paused
+                    break;
+                case 3: // Succeeded
+                    button.text = "Reset"
+                    break;
+                case 4: // Failed
+                    button.text = "Export area"
+                    break;
+                case 5: // Cancelling
+                    break;
+                default:
+                    break;
+                }
```

Comment:
Same feedback as the other switch statements. Could the cases be enums rather than numbers+comments?